ps_menutoplinks.class.php

```diff
@@ -67,6 +67,16 @@ public static function getLinkLang($id_linksmenutop, $id_shop)
         return array('link' => $link, 'label' => $label, 'new_window' => $new_window);
     }
 
+    public static function updateUrl($link)
+    {
+        for($i = 1; $i <= count($link); $i++) {
+            if (substr($link[$i], 0, 7) !== "http://" && substr($link[$i], 0, 8) !== "https://") {
+                $link[$i] = "http://" . $link[$i];
```

Comment:
Hi @aleeks, I don't understand the usefulnesss of using the variable $_SERVER in this context.
I think it's a good idea to use directly "http://" and "https://" for this.